src/Http/Http.Extensions/gen/StaticRouteHandlerModel/EndpointParameter.cs

```diff
@@ -617,7 +617,11 @@ private static string ConvertEndOfLineAndQuotationCharactersToEscapeForm(string
 
     public bool SignatureEquals(object obj) =>
```

Comment:
Signature comparison is a whole thing.  I don't suppose roslyn exposes theirs?

Reply:
Hmmm...not that I am aware of.

Although, FWIW, the signature here is contextualized to the things we care about for minimal APIs in particular (for example they account for the HTTP method associated with a request) so the Roslyn model might not be totally helpful here.